voter-dapp/src/ResolvedRequests.js

```diff
@@ -138,6 +138,13 @@ function ResolvedRequests({ votingAccount }) {
                 secondary={prettyFormatNumber(voteStatsDialogData.correctlyRevealedVotesPct) + "% of Revealed Votes"}
               />
             </ListItem>
+            <ListItem>
+              <ListItemText
+                primary={"Rewards Available: " + prettyFormatNumber(voteStatsDialogData.roundInflationRewardsAvailable)}
+                // I don't use `prettyFormatNumber` here because for some reason it rounds 0.0005 to 0.001
```

Comment:
@chrismaree @mrice32 i can't figure out why `.toLocaleString({ minimumFractionDigits: 4 })` cuts `0.0005` to `0.001`

Reply:
I tried out a few examples here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString. For whatever reason, I don't think this actually does what we think it does. I've looked at the docs, and I can't figure out where the issue is, but for now it's probably better to just avoid it and print the raw number or use toFixed. Note: I might also suggest avoiding it in other places (like the line above) as well since it's not behaving as you expect.

Reply:
@mrice32 the main advantage of `.toLocaleString()` to me is that it puts in comma's every 3 digits :)